docs/References.bib

```diff
@@ -2278,6 +2278,29 @@ @article{Porth2016rfi
   SLACcitation   = "%%CITATION = ARXIV:1611.09720;%%"
 }
 
+@ARTICLE{radice2022,
+       author = {{Radice}, David and {Bernuzzi}, Sebastiano and {Perego},
+       Albino and {Haas}, Roland},
+        title = "{A new moment-based general-relativistic neutrino-radiation
+        transport code: Methods and first applications to neutron star
+        mergers}",
+      journal = {\mnras},
```

Comment:
This isn't displaying correctly, but that seems to be a wider issue with our documentation, so I've filed #6469.  Don't worry about it here unless that gets resolved really quickly before this is merged.